main.c

```diff
@@ -56,6 +56,8 @@
 #include "libtcmu_config.h"
 #include "libtcmu_log.h"
 
+# define TCMU_LOCK_FILE   "/var/run/tcmu.lock"
```

Comment:
Is this more normally in the lock dir like

/var/run/lock/tcmu.lock
or
/var/run/lock/tcmu/lock
?

Reply:
Yes will pick the first one.